bgstally/fleetcarrier.py

```diff
@@ -500,6 +480,6 @@ def _from_dict(self, dict: dict):
         self.onfoot_mats_buying = dict.get('onfoot_mats_buying', [])
         self.commodities_selling = dict.get('commodities_selling', [])
         self.commodities_buying = dict.get('commodities_buying', [])
-        self.cargo = dict.get('cargo', [])
-        self.locker = dict.get('locker', [])
-        self.data = dict.get('data')
\ No newline at end of file
+        self.cargo = dict.get('cargo', {})
```

Comment:
Will this not cause a problem with backward-compatibility, where a user already has lists stored in their JSON?